lesson13/css/style.css

```diff
@@ -1,3 +1,82 @@
+
+body {
+  font-size: 14px;
+  user-select: none;
+}
+
+ul {
+  pointer-events: none;
+}
+
 button {
   cursor: pointer;
+  margin: 10px auto;
+}
+
+.button-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 10px auto;
+}
+
+.open-button {
+  display: block;
+  cursor: pointer;
+  width: 200px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  text-align: center;
+  padding: 12px 0;
+  margin: 16px auto 0;
+  transition: opacity 0.8s;
+}
+
+.mask {
+  background: rgba(0, 0, 0, 0.4);
+  position: fixed;
+  inset: 0;
+  z-index: 1;
+}
+
+.modal, .back-button {
+  justify-content: center;
+  background: #fff;
+  width: 200px;
+  padding: 20px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  position: absolute;
+  top: 40px;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  opacity: 0.8;
+  transition: transform 0.4s;
+  z-index: 2;
+}
+
+.back-button {
+  width: 100px;
+  text-align: center;
+  padding: 12px 0;
+  margin: 16px auto 0;
+}
```

Comment:
> When I remove "position: absolute;", the button moves to the bottom of the window.
I can't fix it ;<
Please tell me how to fix it.
https://github.com/groovywave/JS_Practice/pull/36/files#r1174222774


やりたいことと違ったらすみません🙏参考コードを書きました。

.back-buttonにも`absolute`をつけてしまうと、画面幅を縮めた時にリストに被ってしまいます。
![Screenshot at Apr 22 11-42-13](https://user-images.githubusercontent.com/67426438/233758082-a82249db-1a40-4d96-bbbe-a6b427ca6b60.png)

普通にmargin制御にしました。これではどうでしょうか？
[CodeSandbox](https://codesandbox.io/s/jske-ti-13-can-kao-kodo-qi10w2)

```suggestion
.modal {
  justify-content: center;
  background: #fff;
  width: 200px;
  padding: 20px;
  border: 1px solid #ccc;
  border-radius: 4px;
  position: absolute;
  top: 40px;
  left: 0;
  right: 0;
  margin: 0 auto;
  opacity: 0.8;
  transition: transform 0.4s;
  z-index: 2;
}

.back-button {
  display: block;
  width: 100px;
  text-align: center;
  padding: 12px 0;
  margin: 16px auto 0 auto;
  transition: transform 0.4s;
}
```

こんな感じになります。
![Screenshot at Apr 22 11-51-56](https://user-images.githubusercontent.com/67426438/233758146-aa55e898-4fc0-4cfb-97bf-27b75360a0bf.png)

Reply:
I fixed that:)
a9cf824aaa496b5d3f5c2ae6284720ab1d425c26